model_compression_toolkit/core/keras/kpi_data_facade.py

```diff
@@ -68,7 +68,7 @@ def keras_kpi_data(in_model: Model,
             Create a random dataset generator:
 
             >>> import numpy as np
-            >>> def repr_datagen(): return [np.random.random((1,224,224,3))]
+            >>> def repr_datagen(): yield [np.random.random((1, 224, 224, 3))]
```

Comment:
maybe add a comment that this API only makes one call for the generator and that's why there is no need for an external loop?